CalibTracker/SiStripHitEfficiency/src/HitEff.cc

```diff
@@ -101,13 +101,25 @@ void HitEff::beginJob(){
   edm::Service<TFileService> fs;
 
   traj = fs->make<TTree>("traj","tree of trajectory positions");
+  #ifdef CALIBTreeDEV
   traj->Branch("TrajGlbX",&TrajGlbX,"TrajGlbX/F");
   traj->Branch("TrajGlbY",&TrajGlbY,"TrajGlbY/F");
   traj->Branch("TrajGlbZ",&TrajGlbZ,"TrajGlbZ/F");
-  traj->Branch("TrajLocX",&TrajLocX,"TrajLocX/F");
-  traj->Branch("TrajLocY",&TrajLocY,"TrajLocY/F");
+  traj->Branch("timeDT",&timeDT,"timeDT/F");
+  traj->Branch("timeDTErr",&timeDTErr,"timeDTErr/F");
+  traj->Branch("timeDTDOF",&timeDTDOF,"timeDTDOF/I");
+  traj->Branch("timeECAL",&timeECAL,"timeECAL/F");
+  traj->Branch("dedx",&dedx,"dedx/F");
+  traj->Branch("dedxNOM",&dedxNOM,"dedxNOM/I"); 
   traj->Branch("TrajLocErrX",&TrajLocErrX,"TrajLocErrX/F");
   traj->Branch("TrajLocErrY",&TrajLocErrY,"TrajLocErrY/F");
+  traj->Branch("istep",&istep,"istep/I");
+  traj->Branch("nLostHits",&nLostHits,"nLostHits/I");
+  traj->Branch("chi2",&chi2,"chi2/F");
+  traj->Branch("p",&p,"p/F");
+  #endif
+  traj->Branch("TrajLocX",&TrajLocX,"TrajLocX/F");
+  traj->Branch("TrajLocY",&TrajLocY,"TrajLocY/F");
   traj->Branch("TrajLocAngleX",&TrajLocAngleX,"TrajLocAngleX/F");
```

Comment:
TrajLocAngleX and TrajLocAngleY could be removed from the tree. They are not used at all in the hit efficiency codes.
@mmusich do you want to keep them for other studies that could profit from the hitEff tree ?

Reply:
@jlagram, indeed. We need them to compute the effective silicon thickness to correct S/N in Olivier's analysis